### ISSUE TITLE
Song remains loading

### ISSUE THREAD

**Issue by f3d3r1c098** [steve228uk/YouTube-Music#165]
when trying to play a song or playlist it never loads
<img width="1304" alt="Captura de Pantalla 2022-05-02 a la(s) 22 08 14" src="https://user-images.githubusercontent.com/104804711/166390937-a0f103a8-abfd-4725-b4c5-82ea7e85224a.png">

**Comment by f3d3r1c098** [comment]
What is NaN:NaN?
![Uploading Captura de Pantalla 2022-05-02 a la(s) 22.33.41.png…]()

**Comment by Ilanss** [comment]
I've got the same issue since the last update today.

<img width="378" alt="Screenshot 2022-05-03 at 09 11 40" src="https://user-images.githubusercontent.com/12672596/166416234-4698e8e1-e408-462f-9e86-f1eda38bbc25.png">

**Comment by mark-pictor-csec** [comment]
@Ilanss weird, 1.3.1 is the first version that didn't work for me. 

I don't even get the playback controls bar at the bottom, so I'm missing out on the Not a Number fun 😢

**Comment by abesmon** [comment]
same problem here. 1.3.2 - infinite loading. Blank screen after logout. Tried to reinstall - no progress :(

**Comment by aquaticone** [comment]
I'm having this same issue. FWIW I thought this was an abandoned project and was still on version 1.0.8 where this also started happening. That led me to seek out this repo and install the latest version -- still happening on 1.3.2.

Perhaps an issue with the Google APIs?

**Comment by Ilanss** [comment]
Did the 1.3.3 update and no problem as up for now :)

PS: also did a MacOs update yesterday

**Comment by mark-pictor-csec** [comment]
Updated to 1.3.3, but it just gives me the spinning progress indicator... 😞 

Are there any logs I can pull that'd be handy? If so, where would I find them?

Can the service workers be stopped from the cli? El goog wasn't terribly helpful on that, just gave me browser-specific info.

**Comment by jscheel** [comment]
After 1.3.3 is installed, try restarting the app. See if that kicks things into gear.

**Comment by mark-pictor-csec** [comment]
Between the point where 1.3.3 didn't work and did work, I applied an OS update and rebooted. One would hope that neither of those is required, but worst case @alkalim just wait for the next os update 😉

**Comment by abesmon** [comment]
agreed! I've updates OS and rebooted today, and now YTMusic working fine

**Comment by alkalim** [comment]
Strangely enough, the sequence of restarting the mac - updating macOS - restarting the mac again - reinstalling 1.3.3 finally got it working. Thanks for the advice!
